hedera-node/docs/design/services/smart-contract-service/udpate_hts_smart_contract.md

```diff
@@ -0,0 +1,86 @@
+# Update Hedera Token Service Smart Contract
+
+## Purpose
+
+In order to support updating the Token Custom Fees via smart contract calls, we need to expose a new function in the Hedera Token Service Smart Contract. This document will define the architecture and implementation of the `tokenFeeScheduleUpdate` function that will extend tha capabilities of the Hedera Token Service Smart Contract.
+
+## References
+
+[HIP-514](https://hips.hedera.com/hip/hip-514) - HIP that introduces token management via smart contracts.\
```

Comment:
Reference [HIP-18](https://hips.hedera.com/hip/hip-18) "Custom Hedera Token Service Fees" as well, since that's where the HAPI transaction "TokenFeeScheduleUpdate" is defined?